docs/docs/fallback-handoff.mdx

```diff
@@ -283,6 +285,44 @@ rules:
   - active_loop: action_two_stage_fallback
 ```
 
+### 4. Defining an ultimate fallback action (optional)
```

Comment:
```suggestion
### 4. Defining an ultimate fallback action
```

I would not mark this optional because a silent ending to fallback isn't the intended default behaviour